hnn_qt5.py

```diff
@@ -218,71 +231,174 @@ def runsim (self):
     print(''); self.updatewaitsimwin('')
   
   def optmodel (self):
+    import simdat
+    simdat.initial_ddat = simdat.ddat.copy()
+
     self.updatewaitsimwin('Optimizing model. . .')
-    from neuron import h # for praxis
+
+    self.last_step = False
+    self.first_step = True
+    for step in dconf['opt_info'].keys():
```

Comment:
dconf['opt_info'] is the "global" optimization information (for all steps). self.opt_params gets reset for each step